client/App.tsx

```diff
@@ -1,65 +1,163 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import settings from '../public/settings.png';
 import intime from '../public/intime.png';
 import data from '../public/data.png';
 import barchart from '../public/barchart.png';
 import snippet from '../public/code-snippet.png';
+import logo from '../public/logo.png';
```

Comment:
Rename the file in the public folder from `Logo.PNG` to `logo.png`. Can you also remove any images that aren't being used?

Reply:
This is probably the reason the build is failing.

Reply:
Done!